src/css/components/Button.scss

```diff
@@ -285,67 +135,12 @@ $Button--large-padding: 0 scaleGrid(3);
 $Button--huge-font-size: $tu-large-fontSize;
 $Button--huge-height: scaleGrid(10);
 $Button--huge-line-height: $Button--huge-height - 3;
-$Button--huge-padding: 0 scaleGrid(8);
+$Button--huge-padding: 0 scaleGrid(2);
 
 .Button--huge {
```

Comment:
I think we can ditch this guy altogether. @l3aconator are we still using this behemoth?

Reply:
Bacon is going to move this into the marketing repo if they want to keep using it, so we can remove it for now from TS-UI.